lib/kinde_sdk/kinde_client_sdk.ex

```diff
@@ -426,6 +354,26 @@ defmodule KindeClientSDK do
     throw("Please provide correct grant_type")
   end
 
+  defp fetch_token_with_expiring_timestamps(conn, expiring_timestamp, form_params, client) do
```

Comment:
Documentation can be added for private functions just like for public functions. This allows you to clearly describe the purpose and usage of private functions, making the codebase easier to understand and maintain for anyone working with it.